docs: some updates for PriceAuthority (#383)
Agoric/documentation: main/getting-started/agoric-cli-guide.md

#### Usage

Develop your Dapp as described in the [Agoric Getting Started Guide](/getting-started/).
After using `agoric start` to test with the simulated chain, stop the simulated
chain with Control-C.

> You will need to compile the Golang dependencies in the Agoric SDK.  Run `cd

Reviewer: ```suggestion
You must compile the Golang dependencies in the Agoric SDK.  Run `cd
```
